### ISSUE TITLE
TensorFlow version problem

### ISSUE THREAD

**Issue by mabodx** [jfkirk/tensorrec#139]
Here is the code I copy from the sample:
https://colab.research.google.com/drive/1VPBOMyc9GEUknlK1hgflLHtquaLK3IRg

I got the following error. I think it is the problem with the tensorflow version, the version on the colab is 1.13.0 I think
```/usr/local/lib/python3.6/dist-packages/tensorflow/python/ops/gradients_impl.py:110: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-beee77fbc4f6> in <module>()
     19 
     20 # # Fit the model for 5 epochs
---> 21 model.fit(interactions, user_features, item_features, epochs=5, verbose=True)
     22 
     23 # Predict scores and ranks for all users and all items

/usr/local/lib/python3.6/dist-packages/tensorrec/tensorrec.py in fit(self, interactions, user_features, item_features, epochs, learning_rate, alpha, verbose, user_batch_size, n_sampled_items)
    535                          verbose=verbose,
    536                          user_batch_size=user_batch_size,
--> 537                          n_sampled_items=n_sampled_items)
    538 
    539     def fit_partial(self, interactions, user_features, item_features, epochs=1, learning_rate=0.1,

/usr/local/lib/python3.6/dist-packages/tensorrec/tensorrec.py in fit_partial(self, interactions, user_features, item_features, epochs, learning_rate, alpha, verbose, user_batch_size, n_sampled_items)
    603             # Numbers of features are either learned at fit time from the shape of these two matrices or specified at
    604             # TensorRec construction and cannot be changed.
--> 605             self._build_tf_graph(n_user_features=n_user_features, n_item_features=n_item_features)
    606             session.run(tf.global_variables_initializer())
    607 

/usr/local/lib/python3.6/dist-packages/tensorrec/tensorrec.py in _build_tf_graph(self, n_user_features, n_item_features)
    490 
    491         # Record the new node names
--> 492         self._record_graph_hook_names()
    493 
    494     def fit(self, interactions, user_features, item_features, epochs=100, learning_rate=0.1, alpha=0.00001,

/usr/local/lib/python3.6/dist-packages/tensorrec/tensorrec.py in _record_graph_hook_names(self)
    176             hook = self.__getattribute__(graph_iterator_hook_attr_name)
    177             iterator_resource_name = hook._iterator_resource.name
--> 178             output_types = hook._output_types
    179             output_shapes = hook._output_shapes
    180             output_classes = hook._output_classes

AttributeError: 'Iterator' object has no attribute '_output_types'```

**Comment by jfkirk** [comment]
Hey @mabodx -- could you confirm what TF version Colab is using? TF 1.13.0 only came out as release candidates (https://github.com/tensorflow/tensorflow/releases), and TF 1.13.1 should be working (https://github.com/jfkirk/tensorrec/blob/master/.travis.yml). Once we know the version, we can chase down the bug.

**Comment by mabodx** [comment]
you can see my screenshot and the code in my codelab here is the link.
https://colab.research.google.com/drive/1VPBOMyc9GEUknlK1hgflLHtquaLK3IRg

![image](https://user-images.githubusercontent.com/4045116/55348206-7ef91e00-546b-11e9-8d5b-d4a03620c24b.png)

**Comment by jfkirk** [comment]
Hey @mabodx -- I just realized that a fix has already been merged for this: https://github.com/jfkirk/tensorrec/pull/136

The current `master` version should work, but it has not yet been released. I will do a batch release and that will fix the issue for you.

Thanks!

**Comment by mabodx** [comment]
@jfkirk Sure, Thanks so where can I check the new release?

**Comment by mabodx** [comment]
Currently I am use `tensorrec-0.26.1`

**Comment by jfkirk** [comment]
I've just released 0.26.2, which should include the fix. Try updating and let me know what you find.

**Comment by mabodx** [comment]
Sure be good. thanks
